digital_asset_types/src/dao/generated/mod.rs

```diff
@@ -10,6 +10,7 @@ pub mod asset_grouping;
 pub mod asset_v1_account_attachments;
 pub mod backfill_items;
 pub mod cl_audits;
```

Comment:
With the current way the ORM works this is not possible. However I untangled the entire ORM circular dependency and I will upstream that soon. 

So TLDR, it will be fixed in a subsequent PR!